src/mechanics/timeIntegration/NewmarkDirect.h

```diff
@@ -30,26 +30,11 @@ class NewmarkDirect : public NewmarkBase
     //! @brief constructor
     NewmarkDirect(StructureBase* rStructure);
 
-    void SetMinLineSearchStep(double rMinLineSearchStep)
-    {
-    	mMinLineSearchStep = rMinLineSearchStep;
-    }
-
-    double GetMinLineSearchStep()const
-    {
-    	return mMinLineSearchStep;
-    }
-
     void SetPerformLineSearch(bool rPerformLineSearch)
     {
         mPerformLineSearch = rPerformLineSearch;
     }
 
-    bool GetPerformLineSearch() const
```

Comment:
Why are Get Methods removed? We are currently not using the serialize feature (but at least plan to). In this case, after deserialization (or when a restore with a restart file is used), it would make sense to determine the actual state of these member variables.